app/src/main/java/com/example/parkingsystemjava/mvp/contract/MainActivityContract.java

```diff
@@ -0,0 +1,17 @@
+package com.example.parkingsystemjava.mvp.contract;
+
+public interface MainActivityContract {
```

Comment:
pls rename the name of the contrat to ParkingContract

Reply:
done!